ext/php8/configuration.h

```diff
@@ -4,10 +4,38 @@
 #include <stdbool.h>
 
 #include "compatibility.h"
+#include "ddtrace_config.h"
 #include "ddtrace_string.h"
 #include "env_config.h"
 #include "integrations/integrations.h"
 
+// TODO Tie these into the X Macros below
+static inline bool get_dd_trace_debug(void) {
+    zval *val = zai_config_get_value(DDTRACE_CONFIG_DD_TRACE_DEBUG);
+    return Z_TYPE_P(val) == IS_TRUE;
+}
+
+static inline zai_string_view get_dd_service(void) {
+    zval *val = zai_config_get_value(DDTRACE_CONFIG_DD_SERVICE);
+    return (zai_string_view){.len = Z_STRLEN_P(val), .ptr = Z_STRVAL_P(val)};
+}
+
+static inline zend_array *get_dd_tags(void) {
+    zval *val = zai_config_get_value(DDTRACE_CONFIG_DD_TAGS);
+    GC_ADDREF(Z_ARRVAL_P(val));
```

Comment:
Either addref strings as well, or don't either. I wouldn't addref either. user can incref/destroy if needed.

Reply:
👍 Yes, let's let the caller decide what to do and not addref here.